### PR TITLE
add tf.where(e.g. Select) support with Loop+If

### PR DESCRIPTION
**Pull request by pengwa**

### CODE REVIEW

**Comment by pengwa** [comment]
@guschmue  please kindly help review this, which would help unblock attention conversion.

File: tf2onnx/tfonnx.py [hunk 1]
```diff
@@ -1536,12 +1547,13 @@ def mknode(op_type, inputs, opname, **kwargs):
     "Relu6": (relu6_op, []),
     "Reshape": (reshape_op, ["Reshape"]),
     "Rsqrt": (rsqrt_op, []),
+    "Select": (select_op, []),
```
Comment (guschmue) [inline tf2onnx/tfonnx.py]: dpesn't this want to go into the opset8 table?

Comment (pengwa) [reply tf2onnx/tfonnx.py]: nice catch! will fix it .

**Comment by pengwa** [comment]
thanks a lot !

**Pull request merged**